crates/nu-command/src/path/join.rs

```diff
@@ -90,6 +90,8 @@ the output of 'path parse' and 'path split' subcommands."#
                 example: r"'C:\Users\viking' | path join spams this_spam.txt",
                 result: Some(Value::test_string(r"C:\Users\viking\spams\this_spam.txt")),
             },
+            // TODO: an example with '..'
+            // TODO: an example with '/'
```

Comment:
Looks like you have these examples below.  Remove TODOs?  😊

Reply:
actually there are two example sections, one for windows and one for linux and macos, i only changed the linux part :yum: